services/weatherService.js

```diff
@@ -0,0 +1,67 @@
+fluid.registerNamespace("myL3");
+
+fluid.defaults("myL3.weatherService", {
+    gradeNames: "fluid.component",
+    invokers: {
+        addDataIntoTable: {
+            funcName: "myL3.weatherService.addDataIntoTable",
+            args: ["{arguments}.0", "{arguments}.1"]
+        }
+    }
+});
+
+myL3.weatherService.addDataIntoTable = function (weatherData, fields) {
+    // Pushing values into values Array using fields array
+    for(let i=0; i<fields.length; i++) {
+        switch(fields[i]) {
+            case "1":
+                myL3.values.push(parseFloat((weatherData.main.temp - 273.15)*1.8 + 32).toFixed(2) +' &deg;F');
```

Comment:
`myL3.values` appears to be local to this function -- there is no need for this global variable

Reply:
Thanks!
Just updated this one.